docs/deploy.md

```diff
@@ -97,35 +108,38 @@ SEE ABOVE FOR WARNINGS ABOUT USING GIT FOR THIS.
 
 # Deploying a prepared cluster
 
-## Cloning existing cluster configuration
-
-Skip this step if you're starting a new cluster.
+## Cloning an existing cluster configuration
 
 To download existing configuration:
 
     $ export HOMEWORLD_DIR="$HOME/my-cluster"
     $ export HOMEWORLD_DISASTER="/media/usb-crypt/homeworld-disaster"
-    $ git clone git@github.com:sipb/hyades-cluster $HOMEWORLD_DIR
+    $ git clone git@github.mit.edu:sipb/hyades-cluster $HOMEWORLD_DIR
 
 Make sure to verify that you have the correct commit hash, out of band.
 
-## Set SSH configuration
+## Extracting upstream keys
+
+For the development cluster, we already have upstream keys. Extract them from the configuration like so:
+
+    $ spire keytab export egg-sandwich egg-keytab
+    $ spire https export homeworld.mit.edu ./homeworld.mit.edu.key ./homeworld.mit.edu.pem
+
+# Configuring SSH
 
 Configure SSH so that it has the correct certificate authority in ~/.ssh/known_hosts for members of the cluster:
 
     $ spire access update-known-hosts
 
-## Building the ISO
+# Building the ISO
 
 Now, create an ISO:
 
     $ spire iso gen preseeded.iso ~/.ssh/id_rsa.pub   # this SSH key is used for direct access during cluster setup
 
-Now you should burn and/or upload preseeded.iso that you've just gotten, so
-that you can use it for installing servers. Make a note of the password it
-generated.
+Now you should burn and/or upload preseeded.iso that you've just gotten, so that you can use it for installing servers. Make a note of the password it generated.
```

Comment:
We no longer need to note the generated password, since it's stored automatically.